lib/fluent/plugin/out_label_router.rb

```diff
@@ -113,8 +113,8 @@ def filter_select(match, metadata)
           unless match.container_names.empty? || match.container_names.include?(metadata[:container])
             return false
           end
-          # Break if list of namespaces is not empty and does not include actual namespace
-          unless match.namespaces.empty? || match.namespaces.include?(metadata[:namespace])
+          # Break if list of namespaces is not empty and does not containe any entry that match actual namespace
+          unless match.namespaces.empty? || match.namespaces.any? { |pattern| Regexp.new(pattern).match?(metadata[:namespace]) }
```

Comment:
I beleive this could have unwanted performance implications. I would consider inteoducing a new field for this (i.e. namespace_regex)

Reply:
You're right! 
The number of cpu cycles needed by the new code (even without using a regex in the namespace entry) is higher that before.
Your proposal makes total sense.

Reply:
Change alredy done. Thanks for the suggestion!